pkg/eventsink/es.go

```diff
@@ -101,7 +106,7 @@ func (es *EventSink) Stop() error {
 func (es *EventSink) networkEventWorker() error {
 	for event := range es.networkEventChannel {
 		bucket := fmt.Sprintf("network-%s-%s-%s", event.Namespace, event.PodName, event.ContainerID)
-		err := es.fileDB.Update(func(tx *bolt.Tx) error {
+		err := es.fileDB.Update(func(tx *badger.Tx) error {
```

Comment:
These lines result with error in compilation
```
undefined: badger.Tx
```

Can you please run `make build` in the root directory?

Reply:
I have figured out my compilation issues and was able to build this successfully.